plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardFragment.java

```diff
@@ -106,6 +107,7 @@ public boolean hasComposite() {
 	public void performFinish(IProgressMonitor monitor) throws CoreException {
 		if(serverSettingsWizardPage != null) {
 			serverSettingsWizardPage.getModel().updateServer();
+			serverSettingsWizardPage.getModel().getServer().setAttribute(OpenShiftServerUtils.SERVER_START_ON_CREATION, true);
```

Comment:
The goal here was to make it so that not having the attribute set would not start the server. If I simply added this to the model or the OpenShiftServerUtils,  then our test cases would not be able to use these methods or models without then overwriting the value to false. The goal is of course to test what the behavior is when the flag isn't set at all, and the way to do that is to ensure the model itself isn't setting the value.